src/Narato.Correlations.SerilogMiddleware/Middleware/CorrelationSeriLogEnrichingMiddleware.cs

```diff
@@ -23,7 +23,7 @@ public Task Invoke(HttpContext context)
                 throw new Exception("No correlation ID was found on the response headers. Did you set up Narato.Correlations correctly?");
             }
 
-            using (LogContext.PushProperty("CorrelationId", context.Response.Headers[CorrelationIdProvider.CORRELATION_ID_HEADER_NAME].First()))
+            using (LogContext.PushProperty("NarCorrelationId", context.Response.Headers[CorrelationIdProvider.CORRELATION_ID_HEADER_NAME].First()))
```

Comment:
this would break existing projects (not that there are any, but it's still good practice to play by this rule imho)

I was going to whine about this, but I guess it's a stupid effort to turn this property into an option...